procedures/meeting-observers.md

```diff
@@ -0,0 +1,39 @@
+# Meeting observers
+
+## Motivation
+
+There has been some interest in people joining meetings in a pure observational manner. (See [previous discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Observing.20meetings.3F).) Further discussion on the specifics of a proposal happened on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Meeting.20observers) and [in a council meeting](https://github.com/rust-lang/leadership-council/blob/245b22a105546a680aad582cf856311a80e93baf/minutes/sync-meeting/2023-07-20.md).
+
+## Process for meeting observers
+
+### Request for observation
+
+As an initial iteration, any Project member may join the public portion of council meetings to observe. No particular method of communication is required; those interested should either post in the [council zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council), send an email to `council@rust-lang.org`, or (if necessary) message a council member. At least for now, non-Project members will not be admitted as meeting observers.
```

Comment:
We should probably give at least some interim definition of what counts as a project member, since we don't really have a good definition.

I'd suggest something like:

```suggestion
As an initial iteration, any Project member may join the public portion of council meetings to observe. No particular method of communication is required; those interested should either post in the [council zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council), send an email to `council@rust-lang.org`, or (if necessary) message a council member. At least for now, non-Project members will not be admitted as meeting observers.

Note that "Project Membership" is current poorly defined and is one of the Council's ongoing areas of work.
For the purposes of this proposal, we consider a project member to be anyone listed as a member of one of the Teams, Subteams, or Working Groups on the [Rust Governance Page](https://www.rust-lang.org/governance).
In the event that the Council decides on a new definition for Rust project membership, this proposal will use the new definition.
```

Reply:
I don't think this necessarily needs to go into the document. (I did consider adding in "included in the @all list for the teams repo, but that just seemed too pedantic)

Reply:
I think this should go in the document; otherwise we'll probably get requests/questions along the lines of "i'm an alumni of a team, can i come?" or "I'm in the unofficial community discord, that mean's i'm a project member, right?"

Reply:
I added `as currently defined by anyone listed in the 'all' team defined in the team repo`.